helpers/date.js

```diff
@@ -0,0 +1,23 @@
+const moment = require('moment');
```

Comment:
please consider,  using the datetime util for language dependant date formatting and send an isostring of the date to the server instead. https://github.com/hpi-schul-cloud/schulcloud-client/blob/main/static/scripts/datetime/datetime.js

Reply:
we changed to ISO string